protocols/CoverageCalculations_gvcf.sh

```diff
@@ -53,33 +53,33 @@ else
 fi
 ml ngs-utils
 
-INPUTS=()
-for batch in "${batchID[@]}"
-do
-	
-	gVCF="${projectResultsDir}/variants/gVCF/${externalSampleID}.batch-${batch}.variant.calls.g.vcf.gz"
-	if [[ -f "${gVCF}" ]]
-	then
-		array_contains INPUTS "-I ${gVCF}" || INPUTS+=("-I ${gVCF}")
-	fi
-done
-
-if [[ "${#INPUTS[@]:-0}" -eq '0' ]]
-then
-	echo "There are no gVCF files"
-else
-	if [[ ! -f "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz" ]]
-	then
-		gatk GatherVcfs \
-		"${INPUTS[*]}" \
-		--OUTPUT "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz"
-	fi
-	
-	if [[ ! -f "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz.tbi" ]]
-	then
-		tabix -p vcf "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz"
-	fi
-fi
+# INPUTS=()
+# for batch in "${batchID[@]}"
+# do
+#
+# 	gVCF="${projectResultsDir}/variants/gVCF/${externalSampleID}.batch-${batch}.variant.calls.g.vcf.gz"
+# 	if [[ -f "${gVCF}" ]]
+# 	then
+# 		array_contains INPUTS "-I ${gVCF}" || INPUTS+=("-I ${gVCF}")
+# 	fi
+# done
+#
+# if [[ "${#INPUTS[@]:-0}" -eq '0' ]]
+# then
+# 	echo "There are no gVCF files"
+# else
+# 	if [[ ! -f "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz" ]]
+# 	then
+# 		gatk GatherVcfs \
+# 		"${INPUTS[*]}" \
+# 		--OUTPUT "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz"
+# 	fi
+#
+# 	if [[ ! -f "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz.tbi" ]]
+# 	then
+# 		tabix -p vcf "${intermediateDir}/${externalSampleID}.merged.g.vcf.gz"
+# 	fi
+# fi
```

Comment:
leave it in for now, this is when we want to have the option to merge the gvcf files first